pubsub/google/cloud/pubsub_v1/subscriber/consumer.py

```diff
@@ -0,0 +1,266 @@
+# Copyright 2017, Google Inc. All rights reserved.
```

Comment:
make this private for now, we may want to move this into `api.core` at some point.